packages/playground/src/twoslashInlays.ts

```diff
@@ -31,12 +39,15 @@ export const createTwoslashInlayProvider = (sandbox: Sandbox) => {
           // @ts-ignore
           kind: 0,
           position: new sandbox.monaco.Position(endPos.lineNumber, endPos.column + 1),
-          text,
-          whitespaceBefore: true,
+          label: text,
```

Comment:
this PR changed `text` to `label` and also changed the return type of `provideInlayHints`:
https://github.com/microsoft/vscode/commit/3677cc48a854386a9344452fe4965df09cf42c05